src/alire/os_linux/alire-platform.adb

```diff
@@ -41,15 +41,17 @@ package body Alire.Platform is
                   Normalized : constant String :=
                                  To_Lower_Case (Replace (Line, " ", ""));
                begin
-                  if Starts_With (Normalized, "id=") then
+                  if Starts_With (Normalized, "id=") or else
```

Comment:
in my ubuntu `os-release` I have:
```
ID=ubuntu
ID_LIKE=debian
```

Looks like there is a risk of having a bad detection if `ID_LIKE` is first in the file.

Reply:
Why not use `ID` for Arch Linux?

Reply:
Hello Fabien,
I suggested this modification because my Linux distribution is Manjaro which is based on Arch.
In my /etc/os-release there is:
ID=manjaro
ID_LIKE=arch

I agree with you that there is a risk of bad detection.
In this case, I could revert the detection to rely only on ID and add both Arch and Manjaro as known distributions.
What do you suggest ?

Nota : there is the same problem on a linuxmint distribution:
ID=linuxmint
ID_LIKE=ubuntu

Reply:
Depends if there is a chance that we need to have different package names for `manjaro` and `arch`.

Reply:
Maybe we can look at `ID` first, and if not recognize fallback to `ID_LIKE`.